src/components/SlateEditor/EditorFooter.tsx

```diff
@@ -260,6 +282,24 @@ function EditorFooter<T extends FormValues>({
           </Wrapper>
           {saveButton}
         </div>
+        {isArticle && (
+          <AlertModal
+            title={t('form.workflow.addComment.warn')}
+            label={t('form.workflow.addComment.warn')}
+            show={showWarningModal}
+            text={t('form.workflow.addComment.description')}
+            actions={[
+              {
+                text: 'Ok',
+                onClick: () => {
+                  setShowWarningModal(!showWarningModal);
+                  modalShown.current = true;
```

Comment:
Trenger vi egentlig dette? Kan vi ikke bare kalle på onSave herifra? Du kan for eksempel lagre `saveAsNewVersion` som et objekt i show-staten, og bruke det inne i denne funksjonen.

Reply:
Ønsket oppførsel er at man skal se warning-modalen om at man ikke har skrevet kommentar, også kan man legge til en kommentar før man lagrer, så å lagre direkte fra modalen blir litt i mot sin hensikt ..